config.json

```diff
@@ -45,6 +45,18 @@
         "prerequisites": [],
         "status": "active"
       },
+      {
+        "slug": "annalyns-infiltration",
```

Comment:
I noticed that all the concept exercises that don't have a story-based name don't even appear in this config. I'm assuming I should add them and you want them on the website.

Reply:
Yes, we removed them while the V3 spec was in flux, but if they work they should be in the config now. Thank you!